subworkflows/nf-core/vcf_gather_bcftools/nextflow.config

```diff
@@ -0,0 +1,5 @@
+params {
+    withName: "^.*:VCF_GATHER_BCFTOOLS:BCFTOOLS_SORT\$" {
+        ext.prefix = {"${meta.id}.sorted"}
```

Comment:
Is this required? If so I would suggest adding a `if` warning in `script` if you expect a clash (like in modules)

Reply:
Need for`^.` and `\$`?

Reply:
not really, just following some regex best practices :)

Reply:
we tend to use just:
```
    withName: ".*:VCF_GATHER_BCFTOOLS:BCFTOOLS_SORT" {
```

Reply:
allright, I'll fix it!